Firestore/Source/Public/FirebaseFirestore/FIRAggregateQuery.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+#import "FIRAggregateSource.h"
+
+@class FIRAggregateQuerySnapshot;
+@class FIRQuery;
+
+NS_ASSUME_NONNULL_BEGIN
+
+NS_SWIFT_NAME(AggregateQuery)
+@interface FIRAggregateQuery : NSObject
+/** :nodoc: */
+- (id)init __attribute__((unavailable("FIRAggregateQuery cannot be created directly.")));
+
+@property(nonatomic, strong, readonly) FIRQuery *query;
+
+#pragma mark - Retrieving Data
+
+- (void)aggregationWithSource:(FIRAggregateSource)source completion:(void (^)(FIRAggregateQuerySnapshot *_Nullable snapshot, NSError *_Nullable error))completion NS_SWIFT_NAME(get(source:completion:));
```

Comment:
Updated the Swift name...
```suggestion
- (void)aggregationWithSource:(FIRAggregateSource)source 
                                  completion:(void (^)(FIRAggregateQuerySnapshot *_Nullable snapshot, NSError *_Nullable error))completion 
                                  NS_SWIFT_NAME(aggregation(source:completion:));
```